src/components/ToDos/ToDoList/index.js

```diff
@@ -1,13 +1,34 @@
-import {useState} from 'react';
+import {useState, useEffect} from 'react';
 
 import NewToDoButton from '../NewToDoButton';
 import ToDo from '../ToDo';
 import ToDoCardTitle from '../ToDoCardTitle';
+import ToDoFilterSection from '../ToDoFilterSection';
 
 import {ToDoListContainer} from './styled';
 
 export default function ToDoList() {
 	const [toDos, setToDos] = useState([]);
+	const [status, setStatus] = useState('all');
+	const [filteredToDo, setFilteredToDo] = useState([]);
+
+	useEffect(() => {
+		filterChangeHandler();
+	}, [toDos, status]);
+
+	function filterChangeHandler() {
+		switch (status) {
+			case 'completed':
+				setFilteredToDo(toDos.filter(todo => todo.completed === true));
+				break;
+			case 'uncompleted':
+				setFilteredToDo(toDos.filter(todo => todo.completed === false));
+				break;
+			default:
+				setFilteredToDo(toDos);
+				break;
+		}
+	}
```

Comment:
You shouldn't need a `useEffect` here. `useEffect`s are for side effects only (localStorage, fetching data). Here is a [good article](https://beta.reactjs.org/learn/you-might-not-need-an-effect) which explains this :) 

I think you could also save yourself the `filteredToDo` state: 
```suggestion
    const filteredToDo = filterChangeHandler();
    
	function filterChangeHandler() {
		switch (status) {
			case 'completed':
				return toDos.filter(todo => todo.completed === true);
				break;
			case 'uncompleted':
				return toDos.filter(todo => todo.completed === false);
				break;
			default:
				return toDos;
				break;
		}
	}
```
Since react will trigger a rerender every time a state is updated, the function should be called every time the status is changed. I hope this helps :)